src/VisionModel/googlenet.jl

```diff
@@ -0,0 +1,59 @@
+using Flux
+
+export googlenet
+
+function conv_block(kernelsize::Tuple{Int64,Int64}, inplanes::Int64, outplanes::Int64; stride::Int64=1, pad::Int64=0)
+  conv_layer = []
+  push!(conv_layer, Conv(kernelsize, inplanes => outplanes, stride = stride, pad = pad)) 
+  push!(conv_layer, BatchNorm(outplanes, relu))
+  return conv_layer
+end
+
+function inceptionblock(inplanes, out_1x1, red_3x3, out_3x3, red_5x5, out_5x5, pool_proj)
+  branch1 = Chain(conv_block((1,1), inplanes, out_1x1)...)
+
+  branch2 = Chain(conv_block((1,1), inplanes, red_3x3)...,
+             conv_block((3,3), red_3x3, out_3x3; pad=1)...)        
+
+  branch3 = Chain(conv_block((1,1), inplanes, red_5x5)...,
+             conv_block((5,5), red_5x5, out_5x5; pad=1)...) 
```

Comment:
It's because the 3x3 and 5x5 aren't meant to reduce the dimensions. For 5x5, the `pad = 2` in order to keep the same dimensions.

Reply:
yes, its working fine now.

Reply:
Okay as soon as you push and the checks pass, we can merge.